release_build_files/readme.md

```diff
@@ -629,6 +629,7 @@ code.
 ## Release Notes
 ### Upcoming Release
 -   Changes
+    - General: Add FirebaseApp.GetApps(), to return the list of firebase app.
     - Firestore: Add support for disjunctions in queries (OR queries)
```

Comment:
This code is already set up to use the emulator with environment variable set, are you sure we shouldn't add it to the readme already?

Reply:
as discussed, temporarily not adding it to release note